xbmc/InfoScanner.cpp

```diff
@@ -25,7 +25,14 @@
 #include "utils/log.h"
 #include "utils/URIUtils.h"
 
-CInfoScanner::~CInfoScanner() = default;
+CInfoScanner::CInfoScanner() :
+  m_showDialog(false),
+  m_handle(nullptr),
+  m_bRunning(false),
+  m_bCanInterrupt(false),
+  m_bClean(false)
```

Comment:
These can be initialized in the header since c++11

Reply:
yes. i was the one who pointed that out to you remember? it's hard to be consistent here since some want it, some want to stick even stuff like the = default implementation of constructors in cpp files..

Reply:
moved to header.

Reply:
For future reference, we tend to stick the : and , at the start of the line to keep alignment.

Sticking =default in cpp file has its use, it solves some issues with unique_ptr for example complaining about dtor not being available. 

Personally I prefer it in the header except for the above exception.